debian/tron.upstart

```diff
@@ -0,0 +1,14 @@
+description "trond"
+
+start on (started networking)
+stop on shutdown
+
+respawn
+
+script
+  set -a
+  if [ -f /etc/default/tron ] ; then
+      . /etc/default/tron
+  fi
+  exec start-stop-daemon --start -c $DAEMONUSER --exec /usr/bin/trond -- --nodaemon $DAEMON_OPTS
```

Comment:
We use modern versions of upstart, can you use 
http://upstart.ubuntu.com/cookbook/#setuid ?

maybe not due to the variables

Reply:
yeah, I couldn't figure out a way to squeeze DAEMONUSER into setuid.